### PR TITLE
Sanderrr/evm 31 deploy tablelandtables to arbitrum nova

### PR DESCRIPTION
**Pull request by sanderpick**
Adds Arbitrum Nova deployment. This should go in after https://github.com/tablelandnetwork/evm-tableland/pull/354.

### CODE REVIEW

File: scripts/verify.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { run, ethers, upgrades, network, proxy } from "hardhat";
```
Comment (joewagner) [inline scripts/verify.ts:1]: Good catch!

**Review by joewagner — APPROVED** [review]
Looks good, I changed the target branch to the branch with #354 so I could look at the diff more easily.  I _think_ if you merge #354, this PR's target will automatically change it's self back to `main`, but if not we should make sure to point this at `main` manually.

File: scripts/verify.ts [hunk 2]
```diff
@@ -8,13 +8,10 @@ async function main() {
     throw Error(`no proxy entry for '${network.name}'`);
   }
 
-  // Verify implementation
-  const tables = (await ethers.getContractFactory("TablelandTables")).attach(
-    proxy
-  );
-  const impl = await upgrades.erc1967.getImplementationAddress(tables.address);
+  // Verify proxy and implementation
+  // This will also link the proxy to the implementation in the explorer UI
```
Comment (sanderpick) [inline scripts/verify.ts:12]: I should have mentioned, this is new functionality in the verify method. no longer need to verify the impl and like it to the proxy using the block explorer UIs.

**Comment by avichalp** [comment]
Looks good 👍

File: .env.example [hunk 3]
```diff
@@ -5,6 +5,8 @@ OPTIMISM_PRIVATE_KEY=0xfixme
 OPTIMISM_API_KEY=fixme
 ARBITRUM_PRIVATE_KEY=0xfixme
 ARBITRUM_API_KEY=fixme
+ARBITRUM_NOVA_PRIVATE_KEY=0xfixme
```
[the remark this answers is not in the document]
Comment (sanderpick) [reply .env.example:8]: Yep!

**Pull request merged**